example/lib/main.dart

```diff
@@ -99,17 +105,36 @@ class _MyHomePageState extends State<MyHomePage> {
               },
             ),
             TextButton(
-              child: const Text('Get feedback'),
+              child: const Text('Provide feedback'),
               onPressed: () {
                 BetterFeedback.of(context)?.show(
                   (
                     String feedbackText,
                     Uint8List? feedbackScreenshot,
-                  ) {
+                  ) async {
                     // upload to server, share whatever
-                    // for example purposes just show it to the user
-                    alertFeedbackFunction(
-                        context, feedbackText, feedbackScreenshot);
+                    // example below: draft an email and send to yourself
```

Comment:
I would prefer if you could add another button. So one button opens the dialog and the other button opens your email example. Then you could hide the button on all unsupported platforms.

Reply:
Sure, got it